### ISSUE TITLE
We have it!

### ISSUE THREAD

**Issue by soffer** [getsenic/nuimo-windows#18]
Guys - we've worked for the last few months to build a simple nuimo-windows app and we have it!
I'll upload it and send you guys a link and will provide a short video as well.
I can tell you that it's working and allows for mapping of the different nuimo actions to keyboard and mouse actions on a Win machine.

Will to try to upload soon.

**Comment by SeidChr** [comment]
I had one ages back but with hacks to use bt api of apps in not-app programms to be able to act as a drive in windows. didnt like that at all, so did ms finally release a propper bluetooth api?

**Comment by soffer** [comment]
Nope... We're using Zadig to address this.
Since I'm not a pro here - where should I upload the build and its instructions?

**Comment by hansmbakker** [comment]
The UWP Bluetooth APIs should be available to win32 applications as well, refer to https://blogs.windows.com/buildingapps/2017/01/25/calling-windows-10-apis-desktop-application/ and https://docs.microsoft.com/en-us/windows/uwp/devices-sensors/bluetooth-dev-faq#can-i-access-these-apis-from-win32 for that.

(I don’t know the specifics of your integration, so maybe you found a blocking limitation there?)

**Comment by Doctan** [comment]
Hi Soffer do how can I get the Win App? Did you upload it somewhere?

**Comment by soffer** [comment]
Let me know where and how and I'll gladly upload

**Comment by Doctan** [comment]
Can you upload it to Rapidshare?

**Comment by beidem** [comment]
Any progress on this?

**Comment by NeilvdM** [comment]
I would love to be able to use the Nuimo windows or my Kodi box.
Did anyone get a finished working app?

**Comment by soffer** [comment]
Let me know how and where to upload it, and it's all yours :)


[image: Mailtrack]
<https://mailtrack.io?utm_source=gmail&utm_medium=signature&utm_campaign=signaturevirality5&>
Sender
notified by
Mailtrack
<https://mailtrack.io?utm_source=gmail&utm_medium=signature&utm_campaign=signaturevirality5&>
11/22/18,
9:55:21 AM

**Comment by NeilvdM** [comment]
Great!,
can you upload it to Wetransfer?
my email is neilvdm@gmail.com

Thanks a mil!

**Comment by hansmbakker** [comment]
@soffer @NeilvdM isn’t it an idea to create a repository for it on GitHub?

**Comment by soffer** [comment]
Yup. Since I’m not sure how to do this, I’ll upload to we transfer and you
can create one

-- 
Adi Soffer

**Comment by NeilvdM** [comment]
once you have uploaded, send me the link and i'll create one

**Comment by soffer** [comment]
Great. Will do.

-- 
Adi Soffer

**Comment by NeilvdM** [comment]
Hi @soffer 
did you managed to upload to wetransfer yet?

**Comment by soffer** [comment]
Uploading now.
Will send the link later today


[image: Mailtrack]
<https://mailtrack.io?utm_source=gmail&utm_medium=signature&utm_campaign=signaturevirality5&>
Sender
notified by
Mailtrack
<https://mailtrack.io?utm_source=gmail&utm_medium=signature&utm_campaign=signaturevirality5&>
11/27/18,
9:31:31 AM

**Comment by soffer** [comment]
https://we.tl/t-VhTc8iBNKh



[image: Mailtrack]
<https://mailtrack.io?utm_source=gmail&utm_medium=signature&utm_campaign=signaturevirality5&>
Sender
notified by
Mailtrack
<https://mailtrack.io?utm_source=gmail&utm_medium=signature&utm_campaign=signaturevirality5&>
11/27/18,
11:42:25 AM

**Comment by NeilvdM** [comment]
thanks, downloading now :)

**Comment by tatemarlow** [comment]
Any chance you can upload to wetransfer again @soffer or @NeilvdM the link is expired!

**Comment by NeilvdM** [comment]
> Any chance you can upload to wetransfer again @soffer or @NeilvdM the link is expired!

Sure! https://we.tl/t-XBoCnUU56r 
Let me know if you get it working

**Comment by tatemarlow** [comment]
Thanks @NeilvdM I take it you had no luck?

**Comment by soffer** [comment]
Hey guys sorry for late reply.
Do you need me to upload again?

-- 
Adi Soffer

**Comment by ghost** [comment]
Hi guys,
I've just seen you created it. Could you reupload @soffer @NeilvdM?

Thanks

**Comment by NeilvdM** [comment]
> Thanks @NeilvdM I take it you had no luck?

Well, Im not much of a Techie when it comes to this .. so I tried once, got stuck and left it there ..

**Comment by NeilvdM** [comment]
> Hi guys,
> I've just seen you created it. Could you reupload @soffer @NeilvdM?
> 
> Thanks

https://we.tl/t-KC6tV9bAZx 
here you go .. let me know if you get it running

**Comment by craigdent** [comment]
Where about can you download the windows app?

Or is there a way to integrate the nuimo using the hub into home assistant?

**Comment by soffer** [comment]
Sorry guys, was away. I’ll try and upload in the coming days

[Adi Soffer - Contact Using Spike](https://www.spikenow.com/?ref=spike-organic-signature&_ts=prsyfb)

On May 20, 2019 at 7:31 GMT, craigdent <notifications@github.com> wrote:

Where about can you download the windows app?

Or is there a way to integrate the nuimo using the hub into home assistant?

**Comment by sabinn** [comment]
> 

@soffer sorry I'm way late to the party. I've dusted off my Nuimo and am very interested in the software you have. Any chance you still it and can share?

**Comment by NeilvdM** [comment]
you know .. I have never ever managed to used mine even .. maybe I should look at this software as well

**Comment by soffer** [comment]
Hey guys, I'm sorry to say I have no idea where the code is.
I'll try and have a look around.
This Nuimo knob is a huge disappointment, and is still lying in my drawer.
My wife looks at it every once in a while and asks when will I finally throw it away.
